control/available_plugin.go

```diff
@@ -380,10 +383,10 @@ func (p *apPool) subscribe(taskId string, subType subscriptionType) {
 
 // subscribe adds a subscription to the pool.
```

Comment:
This comment should be updated to say unsubscribe remove a subscription from the pool. This looks like a copy paste error whenever someone added the comments for subscribe and unsubscribe.